routes/web.php

```diff
@@ -51,27 +51,6 @@ function (): void {
             /* Temp Builder 08 (Review) */
             Route::view('builder-08', 'manager/builder-08')->middleware('localOnly')->name('jpb8');
 
-            /* Application (Welcome Mat) */
-            Route::view('application-01', 'applicant/application/01-welcome')->middleware('localOnly')->name('app1');
-            /* Application (Intro Information & Education) */
-            Route::view('application-02', 'applicant/application/02-info-edu')->middleware('localOnly')->name('app2');
-            /* Application (Experience Instruction) */
-            Route::view('application-03', 'applicant/application/03-exp-instructions')->middleware('localOnly')->name('app3');
-            /* Application (Experience) */
-            Route::view('application-04', 'applicant/application/04-exp')->middleware('localOnly')->name('app4');
-            /* Application (Skills Instruction) */
-            Route::view('application-05', 'applicant/application/05-skill-instructions')->middleware('localOnly')->name('app5');
-            /* Application (Skills) */
-            Route::view('application-06', 'applicant/application/06-skill')->middleware('localOnly')->name('app6');
-            /* Application (Questions) */
-            Route::view('application-07', 'applicant/application/07-questions')->middleware('localOnly')->name('app7');
-            /* Application (Review) */
-            Route::view('application-08', 'applicant/application/08-review')->middleware('localOnly')->name('app8');
-            /* Application (Signature & Submission) */
-            Route::view('application-09', 'applicant/application/09-submit')->middleware('localOnly')->name('app9');
-            /* Application (Congrats) */
-            Route::view('application-10', 'applicant/application/10-congrats')->middleware('localOnly')->name('app10');
-
```

Comment:
Don't delete these demo routes yet - they're still useful for now.

Reply:
Those are all duplicates actually haha. I accidentally added them in when fixing conflicts.

Reply:
Oh! haha